src/main.js

```diff
@@ -169,8 +117,12 @@ const getListedFunctionFiles = async function (
   return srcFiles.map((srcFile) => ({ srcFile, name, mainFile, runtime, extension: extname(srcFile) }))
 }
 
-const getSrcFiles = function ({ runtime, stat, mainFile, extension, srcPath, srcDir, pluginsModulesPath }) {
+const getSrcFiles = function ({ runtime, stat, mainFile, extension, srcPath, srcDir, pluginsModulesPath, useEsbuild }) {
   if (runtime === 'js' && extension === '.js') {
```

Comment:
Should this be changed to `runtime === 'js' && extension !== '.zip'`? So it works with `.ts`.

Reply:
As per our chat, I'm temporarily reverting TypeScript support until we're comfortable with how to proceed.